### ISSUE TITLE
[INST] Not installing version 0.9.3.11.1

### ISSUE THREAD

**Issue by AlekseyCanvas** [coderholic/pyradio#262]
**System:**
 - OS: Debian (Ubuntu)
 - **Python** version: 3
 - `pipx` version (if applicable): 1.0.0-1 all
 - `pipx` installation method: Fully isolated
 - **PyRadio** Version: 0.9.3.11.1

**Terminal output**
![Снимок экрана 2024-11-29 144940](https://github.com/user-attachments/assets/db51552d-57b6-4212-97ef-5bb873af5d8a)
**Additional comments**
After step "Extracting PyRadio source code..." nothing doing

**Comment by s-n-g** [comment]
Hi @AlekseyCanvas 

Let me have a look...

**Comment by s-n-g** [comment]
Yeah, my bad...
I forgot to remove some of the debugging code, including an exit command...
I will fix it for the next release

In the meantime, here's what you can do

Open a terminal, and execute:

```
wget https://github.com/coderholic/pyradio/archive/0.9.3.11.1.zip
unzip 0.9.3.11.1.zip
cd pyradio-0.9.3.11.1
./devel/build_install_pyradio -i
```
This will install `pyradio` as expected

After you verify that the installation was successful, you just

```
cd ..
rm 0.9.3.11.1.zip
rm -rf pyradio-0.9.3.11.1
```

Thank you for reporting this!

**Comment by AlekseyCanvas** [comment]
Thank you for help. Glad to help :)

**Comment by s-n-g** [comment]
Fixed in [version 0.9.3.11.2](https://github.com/coderholic/pyradio/releases/tag/0.9.3.11.2)
